crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -702,6 +714,11 @@ pub(crate) fn check_docstring(
         return;
     }
 
+    // Ignore abstract methods.
+    if visibility::is_abstract(&function_def.decorator_list, checker.semantic()) {
+        return;
+    }
```

Comment:
I guess we could enforce _some_ of the rules, e.g., if there's a default implementation that has a raise, and it's not documented? So, only omit the "extraneous" rules?

Reply:
Yeah, I think I'd continue to enforce the rules that say "you returned/raised/yielded something but didn't document it", and I'd skip the ones that say "you documented that it returns/raises/yields something it doesn't". Abstract methods can have implementations that are called using `super()` from concrete methods in subclasses

Reply:
Kk